src/home_robot/home_robot/mapping/voxel/voxel.py

```diff
@@ -850,6 +850,7 @@ def _show_pytorch3d(
             pointcloud_max_points=200_000,
             boxes_plot_together=True,
             boxes_wireframe_width=3,
+            aspectmode="cube",
```

Comment:
?? what does this do? can we add info to docstring?

Reply:
Plotly kwargs explained in plotly lib doc: https://plotly.com/python-api-reference/generated/plotly.graph_objects.Layout.html?highlight=aspectmode

Reply:
I don't want to reproduce the documentation here since it's better to have one canonical place that doesn't go out of sync